app/src/main/java/com/randomappsinc/studentpicker/database/DataSource.java

```diff
@@ -134,6 +134,38 @@ public void renameList(int listId, String newName) {
         close();
     }
 
+    public Map<ListDO, List<NameDO>> getAllListSet() {
+        Map<ListDO, List<NameDO>> allListMap = new HashMap<>();
+        ListDO listDO = new ListDO(-1, "");
```

Comment:
Why do we create a new listDO out here with these initial values? Why not do a local variable inside the for loop? This feels hard to read.